probe-dictionary/explorer.css

```diff
@@ -131,3 +131,8 @@ body {
 .nav-item {
     font-weight: bold;
 }
+th{
+    position: sticky;
+    top: -1px;
```

Comment:
Why `-1`? Would `0` not make more sense, or is it convention?

Reply:
@chutten It is because when I put it 0 there was some visible space left.